README.md

```diff
@@ -1,28 +1,40 @@
 # btcr-DID-method
 
-To support the development of our DID method, we first needed to create a
-command-line application that would generate a txref from a txid.
+To support the development of our DID method, we created two
+command-line applications. The first application, `txid2txref` can
+convert bitcoin transaction ids, or
+[txids](https://bitcoin.org/en/glossary/txid), to
+[txrefs](https://github.com/veleslavs/bips/blob/c83837536d6629f754ce5a88bbe245e0a615e76e/bip-XXXX-Bech32_Encoded_Transaction_Position_References.mediawiki),
+and vice-versa. It uses bitcoind to verify the transactions. The second
+application, `createBtcrDid`, will create a [decentralized identifier
+(DID)](https://w3c-ccg.github.io/did-spec/) on the bitcoin
+blockchain. It also uses bitcoind to submit the transaction needed to
+create the DID.
 
-## txid2txref
 
-The txid2txref application is based on the txid2bech32 utility created by XXX
-during the 2017 BTCR Virtual Hackathon.
+# txid2txref
 
-txid2txref is run as `txid2txref [txid]` and given a valid bitcoin
-transaction id (aka `txid`) will output a txref along with some other
-info. txid2txref will communicate with an instance of bitcoind via RPC
-to retrieve information about that transaction so it can create the `txref`.
+The txid2txref application is based on the [txid2bech32
+app](https://github.com/WebOfTrustInfo/btcr-hackathon) created during
+the 2017 BTCR Virtual Hackathon.
 
-## Building txid2txref
+txid2txref is run as `txid2txref [txid]` and when given a valid `txid`,
+will output a `txref` along with some other info. txid2txref will
+communicate with an instance of bitcoind via RPC to retrieve information
+about that transaction so it can create the `txref`. txid2txref can also
+be run as `txid2txref [txref]` and will output info about the bitcoin
```

Comment:
I like the zip/unzip dual binary solution (or could be a symlink), since the name describes the action.

Reply:
I like it! Symlink idea seems more like something that could be done after the app is "installed" on a host, so I'll have to look into that. Our makefiles support "install" (thanks to cmake) but I haven't tested that at all yet.